From jeffm@suse.com  Thu Oct  8 14:45:44 2009
From: Jeff Mahoney <jeffm@suse.com>
Date: Thu, 08 Oct 2009 17:30:40 -0400
Subject: Staging: rtl8192e: Add #include <linux/vmalloc.h>
To: Greg KH <gregkh@suse.de>
Message-ID: <4ACE5A00.7010903@suse.com>


This driver uses vmalloc but for whatever reason vmalloc.h isn't included
on ppc.

Signed-off-by: Jeff Mahoney <jeffm@suse.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/rtl8192e/r8192E_core.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/staging/rtl8192e/r8192E_core.c
+++ b/drivers/staging/rtl8192e/r8192E_core.c
@@ -46,6 +46,7 @@
 #undef DEBUG_TX_DESC
 
 //#define CONFIG_RTL8192_IO_MAP
+#include <linux/vmalloc.h>
 #include <asm/uaccess.h>
 #include "r8192E_hw.h"
 #include "r8192E.h"
